qa-tests-backend/src/test/groovy/BaseSpecification.groovy

```diff
@@ -402,10 +401,8 @@ class BaseSpecification extends Specification {
         return Env.get("IS_RACE_BUILD", null) == "true" || Env.CI_JOBNAME == "race-condition-tests"
     }
 
-    static Void printlnDated(String msg) {
-        def date = new Date()
-        def sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.US)
-        println "${sdf.format(date)} ${msg}"
+    static void printlnDated(String msg) {
```

Comment:
Perhaps rename or even inline this method?

Reply:
Inlined